code/__DEFINES/jobs.dm

```diff
@@ -37,9 +37,9 @@
 #define QUARTERMASTER	(1<<6)
 #define CARGOTECH		(1<<7)
 #define MINER			(1<<8)
-#define LAWYER			(1<<9)
-#define CHAPLAIN		(1<<10)
-#define CLOWN			(1<<11)
-#define MIME			(1<<12)
-#define ASSISTANT		(1<<13)
-#define BLUESHIELD		(1<<14)
\ No newline at end of file
+#define CHAPLAIN		(1<<9)
+#define CLOWN			(1<<10)
+#define MIME			(1<<11)
+#define ASSISTANT		(1<<12)
+#define BLUESHIELD		(1<<13)
+#define IAA		(1<<14)

```

Comment:
This define will need to go where `LAWYER` was before, otherwise we might screw something up.